src/workerd/jsg/value.h

```diff
@@ -477,6 +478,32 @@ class StringWrapper {
     return wrap(context, creator, value.asPtr());
   }
 
+  template <StringLike T>
+  kj::Maybe<T> tryUnwrap(v8::Local<v8::Context> context, const v8::FastOneByteString& handle, T*) {
+    auto& js = Lock::from(context->GetIsolate());
+    size_t utf8_length = simdutf::utf8_length_from_latin1(handle.data, handle.length);
+    kj::Array<char> buf = kj::heapArray<char>(utf8_length + 1);
+    buf[utf8_length] = '\0';
+    if (utf8_length == handle.length) {
+      memcpy(buf.begin(), handle.data, handle.length);
```

Comment:
Use of `copyFrom(...)` is preferred. We've been slowing trying to move away from `memcpy` since it's not quite as safe.

Reply:
I replaced it with your suggestion. Thanks!